tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -1665,4 +1665,70 @@ public function theAdministratorTriesToAddTheFollowingUsersToAGroupAtOnceUsingTh
 		}
 		$this->addMultipleUsersToGroup($user, $userIds, $groupId, $table);
 	}
+
+	/**
+	 * @When user :user gets details of the group :groupName using the Graph API
+	 *
+	 * @param string $user
+	 * @param string $groupName
+	 *
+	 * @return void
+	 */
+	public function userGetsDetailsOfTheGroupUsingTheGraphApi(string $user, string $groupName): void {
+		$groupId = $this->getGroupIdByName($groupName);
+		$this->getDetailsOfGroup($user, $groupId);
+	}
+
+	/**
+	 * @When user :user gets details of non-existing group using the Graph API
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 */
+	public function userGetsDetailsOfNonExistingGroupUsingTheGraphApi(string $user): void {
+		// generates random group id for non-existing group
+		$groupId = WebDavHelper::generateUUIDv4();
+		$this->getDetailsOfGroup($user, $groupId);
+	}
+
+	/***
+	 * @param string $user
+	 * @param string $groupId
+	 *
+	 * @return void
+	 *
+	 * @throws GuzzleException
+	 */
+	public function getDetailsOfGroup(string $user, string $groupId): void {
+		$credentials = $this->getAdminOrUserCredentials($user);
+
+		$this->featureContext->setResponse(
+			GraphHelper::getGroup(
+				$this->featureContext->getBaseUrl(),
+				$groupId,
+				$this->featureContext->getStepLineRef(),
+				$credentials["username"],
+				$credentials["password"]
+			)
+		);
```

Comment:
I think we should return the response here and set it in the calling function. This makes it easier for this function to be reused. We might not need to set response in every cases when this function is required.
```suggestion
		
			return GraphHelper::getGroup(
				$this->featureContext->getBaseUrl(),
				$groupId,
				$this->featureContext->getStepLineRef(),
				$credentials["username"],
				$credentials["password"]
			);
```

Reply:
We also have a function https://github.com/owncloud/ocis/blob/fe4195d69410c0a13164d96d8d0770bec893995f/tests/acceptance/features/bootstrap/GraphContext.php#LL543C23-L543C23 
Please check if it fulfills what you're looking for